server/lib/renderers/xlsxReport.js

```diff
@@ -0,0 +1,160 @@
+/**
+ * @overview lib/renderers/xlsx
+ *
+ * @description
+ * This library is just a shim to ensure API uniformity with other renderers.
+ * it renders an Excel report(from data.rows that we pass as render function param).
+ * Having the same data structure will help to have less xlsx renderers
+ * @module lib/renderers/xlsx
+ *
+ * @requires excel4node
+ * @requires lodash
+ */
+const xl = require('excel4node');
+const { find, headers, setValue } = require('./xlsx');
+
+const i18n = require('../helpers/translate');
+
+
+exports.render = render;
+exports.find = find;
+exports.extension = '.xlsx';
+exports.headers = headers;
+
+/**
+ * XLSX Render Method
+ *
+ * @function render
+ *
+ * @description
+ * Renders the dataset as an XLSX file
+ *
+ * @param {Object} data   { rows : []}
+ * @param {Object} options { metadata }
+ */
+function render(data, filename, options) {
+
+  // create a new instance of a Workbook class
+  const wb = new xl.Workbook();
+
+  // get a translated worksheet name
+  const worksheetName = t(filename || 'Sheet1');
+
+  // style applied to cells
+  const headerStyle = wb.createStyle({
+    font : {
+      color : '#ffffff',
+      size : 12,
+      bold : true,
+    },
+    fill : {
+      type : 'pattern',
+      patternType : 'solid',
+      fgColor : '2384F5',
+    },
+  });
+
+  const normalFontSize = {
+    font : {
+      size : 10,
+    },
+  };
+
+  const borderInfo = {
+    style : 'thin',
+    color : '#000000',
+  };
+
+  const styleAllBorders = {
+    border : {
+      bottom : borderInfo,
+      left : borderInfo,
+      top : borderInfo,
+      right : borderInfo,
+    },
+  };
+  const center = {
+    alignment : {
+      horizontal : ['center'],
+    },
+  };
+  const right = {
+    alignment : {
+      horizontal : ['right'],
+    },
+  };
+
+  const titleStyle = {
```

Comment:
These styles are used in other renderers, right?  If so, can we move them into a new file `renderers/styles.js` and then import them into each renderer?